src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -314,6 +320,55 @@ export class BrightScriptDebugSession extends BaseDebugSession {
         }
     }
 
+    private async runAutomaticSceneGraphCommands(commands: string[]) {
+        if (commands) {
+            let connection = new SceneGraphDebugCommandController(this.launchConfiguration.host);
+
+            try {
+                await connection.connect();
+                for (let command of this.launchConfiguration.autoRunSgDebugCommands) {
+                    let response: SceneGraphCommandResponse;
+                    switch (command) {
+                        case 'chanperf':
+                            util.log('Enabling Chanperf Tracking');
+                            response = await connection.chanperf({ interval: 1 });
+                            if (!response.error) {
+                                util.log(response.result.rawResponse);
+                            }
+                            break;
+
+                        case 'fpsdisplay':
+                            util.log('Enabling FPS Display');
+                            response = await connection.fpsDisplay('on');
+                            if (!response.error) {
+                                util.log('FPS Display enabled');
```

Comment:
Why the hardcoded response text for this one? Could we move that response into connection.fpsDisplay?

Reply:
This command returns nothing.
![image](https://user-images.githubusercontent.com/9591618/116616822-81acf280-a913-11eb-9f07-086fddd3147c.png)

Reply:
Looking back at this I understand the second question. Moving it into the response of fps display